doc/articles/features/windows-applicationmodel.md

```diff
@@ -13,12 +13,12 @@ uid: Uno.Features.WAM
 
 ### Android
 
-* Due to the fact that version strings can be arbitrary on Android, we first try to parse `PackageInfo.VersionName`, if it fails, we try to use `PackageInfo.LongVersionCode` and if that fails (this is a `long` value while UWP uses `ushort`), we fall back to default to avoid exceptions (1.0.0.0).
+* Due to the fact that version strings can be arbitrary on Android, we first try to parse `PackageInfo.VersionName`, if it fails, we try to use `PackageInfo.LongVersionCode` and if that fails (this is a `long` value while WinUI uses `ushort`), we fall back to default to avoid exceptions (1.0.0.0).
```

Comment:
```suggestion
* Due to the fact that version strings can be arbitrary on Android, we first try to parse `PackageInfo.VersionName`, if it fails, we try to use `PackageInfo.LongVersionCode`, and if that fails (this is a `long` value while WinUI uses `ushort`), we fall back to default to avoid exceptions (1.0.0.0).
```